src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -311,7 +316,6 @@ export const WorkspaceForm = ({
           permissionErrors[i] = i18n.translate('workspace.form.permission.invalidate.group', {
             defaultMessage: 'Invalid user group',
           });
-          continue;
```

Comment:
I am not preferred to remove this continue command here. We need to add this back if more if condition added.